conan/tools/scm/git.py

```diff
@@ -27,6 +28,12 @@ def get_commit(self):
         except Exception as e:
             raise ConanException("Unable to get git commit in '%s': %s" % (self.folder, str(e)))
 
+    def get_version(self):
```

Comment:
This might be a bit confusing, in the past this interface was used to get the Git version, not the repo version.